src/CodeGen/Microsoft.Data.Web.Design.T4.csproj

```diff
@@ -7,7 +7,7 @@
     <ConditionalAPTCA_L2>true</ConditionalAPTCA_L2>
     <DocumentationFile>$(AssemblyName).xml</DocumentationFile>
     <RestorePackages>true</RestorePackages>
-    <TargetFrameworks>net45</TargetFrameworks>
+    <TargetFrameworks>net48</TargetFrameworks>

```

Comment:
Can we get rid of the codegen project in .net 8.0?

Reply:
I think we can. But that could be done in a separate PR. The focus of this PR was to change the target frameworks and I only made the changes necessary to get that to work. Other changes are beyond the scope of what this PR was trying to achieve.